modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/persistence/impl/DDMFormInstanceFinderImpl.java

```diff
@@ -148,15 +157,27 @@ public List<DDMFormInstance> filterFindByC_G_N_D(
 			orderByComparator, true);
 	}
 
+	@Override
+	public List<DDMFormInstance> filterFindByC_G_N_D_P(
```

Comment:
I don't think we need this method, @leticia-maciel. We can call `doFindByC_G_N_D_P_S` directly in line 476.